control_flow.ipynb

```diff
@@ -798,6 +796,59 @@
    "cell_type": "markdown",
    "id": "44",
    "metadata": {},
+   "source": [
+    "#### Exercise: conditionals inside loops\n",
+    "\n",
+    "Complete the function `solution_filter_by_position` below that filters a list of integers, keeping only numbers larger than their position in the list (**1-based index**).\n",
+    "\n",
+    "The filtered numbers should:\n",
+    "\n",
+    "- **Not** contain duplicates\n",
+    "- Be in **ascending** order\n",
+    "\n",
+    "You must use an `if-else` inside a loop.\n",
+    "\n",
+    "Example: `[0, 3, 1, 2]` should return `[3]` because:\n",
```

Comment:
Just a detail but I would like the example to be [0, 4, 1, 3]` should return `[4] to highlight you must be strictly larger.

Reply:
I think it's already written in the text: "keeping only numbers larger than their position in the list"